crontab.yml

```diff
@@ -2,64 +2,66 @@ jobs:
   - name: SimpleLogin growth stats
     command: python /code/cron.py -j stats
     shell: /bin/bash
-    schedule: "0 0 * * *"
```

Comment:
For this one can we keep the same hour so we can compare it to past data?

Reply:
Sure. But 15m should not make a noticeable difference.